manifests/s/SamsungElectronics/SamsungMagician/4.1/SamsungElectronics.SamsungMagician.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.SamsungMagician
+PackageVersion: 4.1

```

```suggestion
PackageVersion: 4.1.0
```
This should match control panel. The version folder name will also need to be updated